.circleci/setup_macos.sh

```diff
@@ -22,7 +22,7 @@ if [ "$BUILD_CONFIG" == "Release" ]; then
 	brew install awscli
 	echo "========> Install sbt"
 	brew install sbt
-	sbt sbtVersion
+	#sbt sbtVersion
```

Comment:
```suggestion
```
since we don't use it we can remove the line I think.

Reply:
Sure.

Reply:
Done.